packages/sku/src/program/commands/init/init.action.ts

```diff
@@ -131,10 +133,18 @@ export const initAction = async (
     },
   };
 
+  const isRepoRoot = rootDir === null || rootDir === initDir;
```

Comment:
`rootDir` will be `null` inside a new repo.

`rootDir` will _not_ equal `initDir` in basically any monorepo situation. I can't think of a situation where `rootDir === initDir`, but this check is here just in case that ever does happen.